front-end/src/renderer/pages/Settings/components/KeysTab.vue

```diff
@@ -513,26 +689,15 @@ watch(isDeletingKey, () => {
       <AppModal v-model:show="isDeleteModalShown" class="common-modal">
         <div class="p-5">
           <div>
-            <i class="bi bi-x-lg cursor-pointer" @click="isDeleteModalShown = false"></i>
+            <i class="bi bi-x-lg cursor-pointer" @click="handleCloseModal"></i>
           </div>
           <div class="text-center">
             <AppCustomIcon :name="'bin'" style="height: 160px" />
           </div>
           <form @submit.prevent="handleDelete">
             <h3 class="text-center text-title text-bold mt-3">Delete key pair</h3>
```

Comment:
The heading `Delete key pair` may be updated to have `s` when multiple keys are going to be deleted